RealmSwift/Tests/TestCase.swift

```diff
@@ -33,6 +33,7 @@ class TestCase: XCTestCase {
         Realm.defaultPath = realmPathForFile("\(realmFilePrefix()).default.realm")
         NSFileManager.defaultManager().createDirectoryAtPath(realmPathForFile(""), withIntermediateDirectories: true, attributes: nil, error: nil)
 
+        internalImplementation().setNumberOfTestIterations(1)
```

Comment:
This should instead do what we ended up doing for obj-c on master (just `autoreleasepool { super.invokeTests()` here and wrap the bodies of each `setUp` and `tearDown` method with  `autoreleasepool {}`).